apps/core/src/hooks/useGetOriginByteKioskContents.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SuiAddress } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+import { useRpcClient } from '../api/RpcClientContext';
+import { useGetOwnedObjects } from './useGetOwnedObjects';
+
+// OriginByte contract address for mainnet (we only support mainnet)
+const ORIGINBYTE_OWNER_TOKEN_ADDRESS =
```

Comment:
Is this a mainnet address? How are we going to handle testnet then?

Reply:
discussed via slack, but have decided to start with mainnet support and can re-visit if we think that support is critical for networks other than mainnet